apps/hosts/migrations/0004_auto_20250422_2255.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.20 on 2025-04-22 22:55
```

Comment:
Also, instead of three migration files, can you please generate one migration file and name it.